src/App.tsx

```diff
@@ -14,7 +14,7 @@ export const App = () => {
       </div>
 
       <div className="sidebar">
-        <FindMovie />
+        <FindMovie updateMovies={setMovies} movies={movies} />
```

Comment:
and create function for it 

```suggestion
        <FindMovie onAdd={setMovies} movies={movies} />
```